### PR TITLE
Prod release

### PR DESCRIPTION
**Pull request by scottyhq**
@TomAugspurger - would love to get the environment variables option on prod. Seems like there is a lot of prometheus related stuff in this as well.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Yeah, happy to do it whenever.

Not sure what's up with the merge conflicts. Are you able to handle them? (probably just take what's on staging). FWIW, on `pangeo-cloud-federation` we're trying out only allowing "rebase and merge", which will enforce a linear commit history on both staging and prod. Might want to do that here after this PR.

**Comment by scottyhq** [comment]
Sounds good. The number of commits is certainly a bit unwieldily, and for these repos we're interested in the PR level rather than the individual commit level!

**Comment by TomAugspurger** [comment]
Yeah, ideally we would have squash-and-merge on staging and
rebase-and-merge on prod, but GitHub doesn't allow per-branch merge
policies (AFAICT).

**Comment by scottyhq** [comment]
> Yeah, ideally we would have squash-and-merge on staging and
rebase-and-merge on prod, but GitHub doesn't allow per-branch merge
policies (AFAICT).

Let;s go ahead and try it next time!

File: .circleci/config.yml [hunk 1]
```diff
@@ -40,6 +40,9 @@ jobs:
             helm repo add pangeo https://pangeo-data.github.io/helm-chart/
             helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
             helm repo add dask-gateway https://dask.org/dask-gateway-helm-repo/
+            helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+            helm repo add grafana https://grafana.github.io/helm-charts
+            helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```
Comment (consideRatio) [inline .circleci/config.yml:45]: This should be replaced with https://charts.helm.sh/stable, the non deprecated location

File: k8s-aws/readme.md [hunk 2]
```diff
@@ -28,6 +28,9 @@ helm version #>3
 helm repo add pangeo https://pangeo-data.github.io/helm-chart/
 helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
 helm repo add dask-gateway https://dask.org/dask-gateway-helm-repo/
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```
Comment (consideRatio) [inline k8s-aws/readme.md:32]: https://charts.helm.sh/stable here as well

**Pull request merged**